src/components/speakers/SpeakerDirectory.tsx

```
@@ -0,0 +1,205 @@
+                        <Loader2 className="absolute right-3 top-1/2 h-4 w-4 -translate-y-1/2 animate-spin text-indigo-500" />
+                    )}
+                </div>
+            </div>
+
+            <div className="grid grid-cols-1 gap-6 md:grid-cols-2">
+                {/* Search Results Column */}
+                <div className="space-y-3">
+                    {searchResults.length === 0 && searchQuery.length >= 2 && !isSearching && (
+                        <p className="text-sm text-slate-400">No speakers found.</p>
+                    )}
+                    {searchResults.map((speaker) => (
```

ESLint

[error] Delete `··········`

(prettier/prettier)